fala/common/footer_views.py

```diff
@@ -25,3 +26,20 @@ class PrivacyView(FooterView, CommonContextMixin, TranslationMixin, TemplateView
 
 class CookiesView(FooterView, CommonContextMixin, TranslationMixin, TemplateView):
     template_name = "adviser/cookies.html"
+
```

Comment:
You could also do this, where you are getting the context from `FooterView` so you just update the context, instead of overriding and building again. 

```
    def get(self, request, **kwargs):
        response = super().get(request, **kwargs)

        context = response.context_data

        referer = request.META.get("HTTP_REFERER", "/")
        host = request.get_host()
        previous_url = referer if url_has_allowed_host_and_scheme(referer, allowed_hosts={host}) else "/"

        context.update({"previous_url": previous_url})

        return self.render_to_response(context)
```